pipelines/upgrade/04-install_proxy.yaml

```diff
@@ -14,7 +14,7 @@
       - "--foreman-proxy-foreman-base-url https://{{ server_fqdn.stdout }}"
       - "--foreman-proxy-oauth-consumer-key {{ oauth_consumer_key.stdout }}"
       - "--foreman-proxy-oauth-consumer-secret {{ oauth_consumer_secret.stdout }}"
-      - "--foreman-proxy-content-parent-fqdn {{ server_fqdn.stdout }}"
+      - "{{ '' if (pipeline_version == 'nightly' or 2.3 is version(forklift_upgrade_version_start, '>')) else ('--foreman-proxy-content-parent-fqdn %s' % server_fqdn.stdout) }}"
```

Comment:
Actually, this will still omit the param on 17-18-nightly upgrades.

Given the upgrade_start will never be nightly, you can just omit (hah) the first part of the condition.